otx/cli/utils/io.py

```diff
@@ -103,6 +103,8 @@ def read_model(model_configuration: ModelConfiguration, path: str, train_dataset
             full_path = os.path.join(os.path.dirname(path), key)
             if os.path.exists(full_path):
                 model_adapters[key] = ModelAdapter(read_binary(full_path))
+            else:
+                model_adapters[key] = ModelAdapter(bytes())
```

Comment:
Could you explain why we need this for tiling?

Reply:
It's the same reason as previous remark: https://github.com/openvinotoolkit/training_extensions/pull/1516